sections/main-addresses.liquid

```diff
@@ -27,92 +36,125 @@
+        </div>
+        <div class="field">
             <input type="text" id="AddressCompanyNew" name="address[company]" value="{{ form.company }}" autocomplete="organization" placeholder="{{ 'customer.addresses.company' | t }}">
-            <label for="AddressCompanyNew">{{ 'customer.addresses.company' | t }}</label>
-          </div>
-          <div class="field">
+          <label for="AddressCompanyNew">{{ 'customer.addresses.company' | t }}</label>
+        </div>
+        <div class="field">
             <input type="text" id="AddressAddress1New" name="address[address1]" value="{{ form.address1 }}" autocomplete="address-line1" placeholder="{{ 'customer.addresses.address1' | t }}">
-            <label for="AddressAddress1New">{{ 'customer.addresses.address1' | t }}</label>
-          </div>
-          <div class="field">
+          <label for="AddressAddress1New">{{ 'customer.addresses.address1' | t }}</label>
+        </div>
+        <div class="field">
             <input type="text" id="AddressAddress2New" name="address[address2]" value="{{ form.address2 }}" autocomplete="address-line2" placeholder="{{ 'customer.addresses.address2' | t }}">
-            <label for="AddressAddress2New">{{ 'customer.addresses.address2' | t }}</label>
-          </div>
-          <div class="field">
+          <label for="AddressAddress2New">{{ 'customer.addresses.address2' | t }}</label>
+        </div>
+        <div class="field">
             <input type="text" id="AddressCityNew" name="address[city]" value="{{ form.city }}" autocomplete="address-level2" placeholder="{{ 'customer.addresses.city' | t }}">
-            <label for="AddressCityNew">{{ 'customer.addresses.city' | t }}</label>
+          <label for="AddressCityNew">{{ 'customer.addresses.city' | t }}</label>
+        </div>
+        <div class="field-country">
+          <label for="AddressCountryNew">{{ 'customer.addresses.country' | t }}</label>
+          <div class="select">
+            <select
+              id="AddressCountryNew"
+              name="address[country]"
+              data-default="{{ form.country }}"
+              autocomplete="country"
+              {% if form.errors contains 'country' %}
+                aria-invalid="true"
+                aria-describedby="AddressCountryNewError"
+              {% endif %}
+            >
+              {{ all_country_option_tags }}
+            </select>
+            <svg aria-hidden="true" focusable="false" viewBox="0 0 10 6">
+              <use href="#icon-caret" />
+            </svg>
           </div>
-          <div>
-            <label for="AddressCountryNew">{{ 'customer.addresses.country' | t }}</label>
-            <div class="select">
-              <select
-                id="AddressCountryNew"
-                name="address[country]"
-                data-default="{{ form.country }}"
-                autocomplete="country"
+        </div>
+        {%- if form.errors contains 'country' -%}
```

Comment:
It doesn't seem to work for me. On my account it selects by default `United States`. So I select the three dashes instead `---` and I can click add and have the original issue 🤔 

[video](https://screenshot.click/21-28-quy1e-12zmq.mp4)